src/LayoutGridCatalog.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import classnames from 'classnames';
+import ComponentCatalogPanel from './ComponentCatalogPanel.js';
+
+import './styles/LayoutGridCatalog.scss';
+
+const LayoutGridCatalog = () => {
+  return (
+    <ComponentCatalogPanel
+      hero={<LayoutGridHero />}
+      title='Layout Grid'
+      description='Menus display a list of choices on a transient sheet of material.'
```

Comment:
Update description to be about layout grid

Reply:
Woops....